source/documentation/tools/rstudio.md

```diff
@@ -0,0 +1,59 @@
+# RStudio
+
+For general guidance in using RStudio, see the [RStudio documentation](https://docs.rstudio.com/).
+
+## RStudio memory issues
+
+RStudio crashes when it runs out of memory. This is because memory is a finite resource, and it's not easy to predict memory usage or exact availability. But if your data is of order of a couple of gigabytes or more, then simply putting it all into a dataframe, or doing processing on it, may mean you run out of memory. For more about memory capacity in the Analytical Platform, and how to work with larger datasets, see the [memory limits](annexes.html#memory-limits) section.
```

Comment:
The annexes.html#memory-limits link seems to be broken when I run locally (seems to go to http://localhost:4567/tools/annexes.html#memory-limits)